api/KeyboardEvent.json

```
@@ -1799,7 +1799,8 @@
               "version_added": true
             },
             "edge": {
-              "version_added": "12"
+              "version_added": null,
+              "notes": "Edge 18 currently implement this property, but only reporting false."
```

A little rewording of the notes.  Can you also put this in the IE compatibility?

```suggestion
              "notes": "Edge implements this property, however the return value is always <code>false</code>.  See <a href='https://developer.microsoft.com/microsoft-edge/platform/issues/17049017/'>bug 17049017</a>."
```